stat/int.go

```diff
@@ -0,0 +1,27 @@
+package stat // import "github.com/influxdata/influxdb/stat"
+
+import "sync"
+
+type Int struct {
+	mu    sync.RWMutex
```

Comment:
I was using this benchmark:

```
func BenchmarkInt_Add(b *testing.B) {
    v := stat.Int{}

    var wg sync.WaitGroup
    for i := 0; i < b.N; i++ {
        wg.Add(1)
        go func() {
            defer wg.Done()
            v.Add(1)
        }()
    }
    wg.Wait()
}
```

Hm, the documentation does seem to say that. That's strange since I was getting a panic on i386 when the unit tests were running using the struct I made earlier. It might not be an issue though since now I've changed to this struct. Although I only changed to this struct for the mutex so I would want to go back to just using atomic everywhere if that was possible.